functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import pypyodbc
+import os
+import azure.functions as func
+import datetime
+import json
+
+# to handle datetime with JSON
+# It serialize datetime by converting it into string
+def default(o):
+  if isinstance(o, (datetime.datetime, datetime.date)):
+    return o.isoformat()
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+  # define the server and database names
+  server = os.environ["ENV_DATABASE_SERVER"]
+  database = os.environ["ENV_DATABASE_NAME"]
+  username = os.environ["ENV_DATABASE_USERNAME"]
+  password = os.environ["ENV_DATABASE_PASSWORD"]
+
+  # define the connection string
+  driver = '{ODBC Driver 17 for SQL Server}'

```

Comment:
You would need to do some research on how to move the hard-coded ODBC Driver to a local environment.